src/rapidata/rapidata_client/order/rapidata_order_builder.py

```diff
@@ -129,8 +131,12 @@ def create(self, submit: bool = True, max_workers: int = 10) -> RapidataOrder:
         if isinstance(
             self._workflow, CompareWorkflow
         ):  # Temporary fix; will be handled by backend in the future
+            assert all(isinstance(item, MultiAsset) for item in self._assets), (
+                "The media paths must be of type MultiAsset for comparison tasks."
+            )
+            media_paths = cast(list[MultiAsset], self._assets)
```

Comment:
This assumes that the multiassets contain only media assets right?

Reply:
yes, atm this is all we need. but again we could just change the checks for that. it would not be a breaking change